messenger.rst

```diff
@@ -712,8 +712,8 @@ to retry them:
     # retry specific messages
     $ php bin/console messenger:failed:retry 20 30 --force
 
-    # remove a message without retrying it
-    $ php bin/console messenger:failed:remove 20
```

Comment:
I would rather keep the example and just add the new one, what do you think?

Reply:
I agree. I've readded it while merging. Thanks.